tests/tests.js

```diff
@@ -59,6 +59,21 @@ describe("htmlbars-inline-precompile", function() {
     }, /placeholders inside a tagged template string are not supported/);
   });
 
+  it("moduleName is supplies to the precompiler", function() {
+    var fullpath = path.join(__dirname, 'fixtures', 'hello-world.js');
+
+    var precompile = function(template, options) {
+      assert.equal(options.moduleName, fullpath);
```

Comment:
Ember uses it internally for hinting at deprecation warnings in a template.  I use it for building CSS modules.

As for the `relativeName` not being relative, it seems that way.  The documentation says the opposite -- so I'm a bit confused.  Lets hold off on merging until that's resolved, we wouldn't want to leak full paths.